plugin/PlayAssetDelivery/src/main/java/com/google/play/core/godot/assetpacks/PlayAssetDelivery.java

```diff
@@ -33,8 +40,23 @@
  */
 public class PlayAssetDelivery extends GodotPlugin {
 
+  private AssetPackManager assetPackManager;
+
+  /**
+   * Helper function that returns an AssetPackManager instance. Initially part of the constructor,
+   * refactored to provide easy access for writing mock tests.
+   *
+   * @param godot
+   * @return
```

Comment:
Similar to @param, you should either add a description of the object being returned or leave out the @return tag altogether.

Reply:
This function was refactored when I removed the override-return-with-mock-object method. I will keep that in mind in upcoming PRs!